resources/ts/Components/Integrations/Detail/CredentialsLegacyAuthConsumers.tsx

```diff
@@ -41,8 +46,14 @@ export const CredentialsLegacyAuthConsumers = ({
           {legacyProdConsumer && (
             <div className="flex flex-col gap-2">
               {status === IntegrationStatus.Active && (
-                <CopyText>{legacyProdConsumer?.apiKey}</CopyText>
+                <div className="flex gap-1 max-md:flex-col max-md:items-start">
+                  <span className="flex items-center whitespace-nowrap">
+                    {t("details.credentials.api_key")}
+                  </span>
+                  <CopyText>{legacyProdConsumer?.apiKey}</CopyText>
```

Comment:
Is the optional chaining necessary?
```suggestion
                  <CopyText>{legacyProdConsumer.apiKey}</CopyText>
```